DevelopersGuide.adoc

```diff
@@ -1673,7 +1673,7 @@ The program will now look like this:
     (dom/div
       (dom/h4 label)
       (dom/ul
-        (map #(ui-person (comp/computed % {:onDelete delete-person})) people)))))
+        (map (fn [p] (ui-person (comp/computed p {:onDelete delete-person}))) people)))))
```

Comment:
Oh, I changed that because in the previous declaration of this function it was the `fn` form and when one considered the changes via `git diff` this line showed up as well.

Reply:
Hmm, I see your reasoning so I'll initiate a revert.